modules/oe_paragraphs_media/config/install/core.entity_view_display.paragraph.oe_text_feature_media.default.yml

```diff
@@ -0,0 +1,52 @@
+uuid: f76c0fe4-6dca-4f09-b0f0-329bac28a976
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.paragraph.oe_text_feature_media.field_oe_media
+    - field.field.paragraph.oe_text_feature_media.field_oe_plain_text_long
+    - field.field.paragraph.oe_text_feature_media.field_oe_text_long
+    - field.field.paragraph.oe_text_feature_media.field_oe_title
+    - paragraphs.paragraphs_type.oe_text_feature_media
+  module:
+    - image
+    - text
+_core:
+  default_config_hash: pumkdD7hKF0U3kuMZK8UCtQvd3jt6o4MDWAfA85KpiA
```

Comment:
We must remove UUID and hash from configuration we ship in the install directory. We don't have to copy from the general site export directory but use config_devel to export selected configurations in the module's config/install directory.